libMagAOX/app/dev/stdMotionStage.hpp

```diff
@@ -297,7 +297,7 @@ template<class derivedT>
 int stdMotionStage<derivedT>::appStartup()
 {
  
-   derived().createStandardIndiNumber( m_indiP_preset, m_presetNotation, 1.0, (double) m_presetNames.size(), 0.0, "%0.3d");
+   derived().createStandardIndiNumber( m_indiP_preset, m_presetNotation, 1.0, (double) m_presetNames.size(), 0.0, "%.4f");
```

Comment:
So, I didn't totally get this. Is this an index into the preset vector or a floating point number?